src/include/stir/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeData.h

```diff
@@ -115,7 +115,20 @@ public PoissonLogLikelihoodWithLinearModelForMean<TargetT>
   //! Listmode pointer
   shared_ptr<CListModeData> list_mode_data_sptr; 
  
-  int current_frame_num;
+  unsigned int current_frame_num;
+
+  //!
+  //! \brief num_events_to_store
+  //! \author Nikos Efthimiou
+  //! \details This is part of some functionality I transfer from lm_to_projdata.
+  //! The total number of events to be *STORED* not *PROCESSED*.
```

Comment:
I don't understand this comment...

Reply:
If we exclude randoms and/or scattered when you might process more events than those stored or used in the reconstruction process.

Reply:
this objective function does not know if scattered events are handled or not. In any case it doesn't "store" events. so, I still don't understand it :-;

Reply:
alright I'll fix this